components/project-list/project-list.jsx

```diff
@@ -88,9 +99,14 @@ export default React.createClass({
     let showViewMoreBtn;
     let searchResult;
 
-    if (this.props.params && this.props.params.search && projects) {
-      // show search result if on search page and search keyword is presented
-      searchResult = (<p>{projects.length} {projects.length > 1 ? `results` : `result`} found for ‘{this.props.params.search}’</p>);
+    if (this.props.params && this.props.params.search) {
+      if (projects) {
+        // Show search result
+        searchResult = (<p>{projects.length} {projects.length > 1 ? `results` : `result`} found for ‘{this.props.params.search}’</p>);
+      } else {
+        // We are still waiting to hear back from Pulse API. Let's show some 'searching' notice in the meantime.
+        searchResult = (<p>Searching for entries that match ‘{this.props.params.search}’...</p>);
```

Comment:
@kristinashu @alanmoo 
`Searching for entries that match ‘some_keyword’...` <= plz help fixing this sentence that I came up with 😂